### PR TITLE
Implement content serving for maven upload

### PR DESCRIPTION
**Pull request by yma88**

### CODE REVIEW

File: src/main/java/org/commonjava/util/sidecar/services/ProxyService.java [hunk 1]
```diff
@@ -103,28 +104,32 @@ void handleConfigChange( String message )
...
     }
 
+    public Uni<Response> doHead( String packageType, String type, String name, String path, HttpServerRequest request )
+                    throws Exception
+    {
+        String contentPath = UrlUtils.buildUrl( CONTENT_REST_BASE_PATH, packageType, type, name, path );
+        return doHead( contentPath, request );
+    }
+
     @GET
```
Comment (ligangty) [inline src/main/java/org/commonjava/util/sidecar/services/ProxyService.java:114]: I think these GET annotations should be removed from service class.

**Review by ligangty — COMMENTED** [review]
Generally LGTM except a tiny issue in comment

File: src/main/java/org/commonjava/util/sidecar/services/ProxyService.java (hunk 1 above)
Comment (yma88) [reply src/main/java/org/commonjava/util/sidecar/services/ProxyService.java]: @ligangty committed, done

**Review by ligangty — APPROVED** [review]
LGTM.

**Pull request merged**